File: 2026-08/src/day_07/prms_181928/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(vector<int> num_list) {
    string odd = "";
    string even = "";

    for (int x : num_list) {
        if (x % 2) {
            odd += to_string(x);
        } else {
            even += to_string(x);
        }
    }

    return stoi(odd) + stoi(even);
}

File: 2026-08/src/day_07/prms_181928/Solution.java
package day_07.prms_181928;

class Solution {
    public int solution(int[] num_list) {
        StringBuilder odd = new StringBuilder();
        StringBuilder even = new StringBuilder();

        for (int x : num_list) {
            if (x % 2 == 1) {
                odd.append(x);
            } else {
                even.append(x);
            }
        }

        return Integer.parseInt(odd.toString()) + Integer.parseInt(even.toString());
    }
}
